app/src/App/FileBrowser.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+namespace Dappur\App;
+
+/**
+ * Copied from https://github.com/vakata/jstree-php-demos/blob/master/filebrowser/index.php
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+
+// Limit To Log Folder
+ini_set('open_basedir', realpath(dirname(__FILE__) . DIRECTORY_SEPARATOR . ".." . DIRECTORY_SEPARATOR . ".." . DIRECTORY_SEPARATOR . "..".DIRECTORY_SEPARATOR."storage".DIRECTORY_SEPARATOR."log".DIRECTORY_SEPARATOR));
+
+class FileBrowser
```

Comment:
__[PHPMD]__

The class FileBrowser has an overall complexity of 74 which is very high. The configured complexity threshold is 50.


<!-- Comment by [Sider](https://sider.review) -->